src/com/amazon/corretto/crypto/provider/Loader.java

```diff
@@ -243,7 +295,7 @@ private static void assertVersionMatch() {
      * /dev/urandom. Clearly this won't work when we start supporting windows systems. We are intentionally taking
      * as few dependencies here as possible.
      */
-    private synchronized static Path createTmpFile(final String prefix, final String suffix) throws IOException {
+    private static synchronized Path createTmpFile(final String prefix, final String suffix) throws IOException {
```

Comment:
can't this still be `static`? your addition on L344 doesn't reference any instance variables.

Reply:
It is still static. I just had the order of `static` and `synchronized` backwards for best Java style.

Reply:
ah, whoops. misread that.